web/src/beta/lib/core/engines/Cesium/Feature/index.tsx

```diff
@@ -79,7 +79,7 @@ const displayConfig: Record<DataType, (keyof typeof components)[] | "auto"> = {
 // Some layer that is delegated data is not computed when layer is updated.
 // Feature's property of delegated data type is calculated when feature is loaded.
 // So in case of delegated data type, to attach property to layer, we need to use normal property before calculated.
-const PICKABLE_APPEARANCE: (keyof AppearanceTypes)[] = ["raster", "3dtiles"];
+const PICKABLE_APPEARANCE: (keyof AppearanceTypes)[] = ["raster", "3dtiles", "heatMap"];
```

Comment:
If you'll make heatMap a delegated data type, It won't support expression like
```
{
  type: "simple",
  data: {
      type: "heatMap"
  },
  heatMap: {
    valueMap: "https://dl.dropboxusercontent.com/scl/fi/cq2d4m71qg0mer2ce1b38/converted_image.png?rlkey=rhojr5xz013275duxsduivkur&dl=0",
    bounds: {
      east: 140.00000000000003,
      north: 36,
      south: 35.33333333333333,
      west: 139,
    },
    width: {
        expression: "1+2"
    }  
  }
}
```

Reply:
I fixed it:pray:
https://github.com/reearth/reearth/pull/835/commits/58e46946ec861e8b279fa2be5aa3cd4f36eaabde